tests/self/se_time.c

```diff
@@ -157,3 +157,26 @@ void selftest_delay(void)
 }
 
 REGISTER_SELF_TEST(delay, se_manual, &selftest_delay)
+
+void selftest_latency(void)
+{
+   struct datetime d;
+   const int iters = 10000;
+   u64 start, duration;
+
+   printk("\n");
+   printk("Clock read-latency self-test\n");
+   printk("---------------------------------------------\n\n");
+
+   start = RDTSC();
+
+   for (int i = 0; i < iters; i++) {
+      hw_read_clock(&d);
+   }
+
+   duration = (RDTSC() - start) / iters;
+
+   printk("Latency: %" PRIu64 "\n", duration);
+}
+
+REGISTER_SELF_TEST(latency, se_short, &selftest_latency)
```

Comment:
Perf tests are never "se_short". It should last several seconds. But, when such tests are run in the cloud the VM can be preempted for a long time if it burns too many CPU cycles and the delays grow almost exponentially. Therefore a test that consume several seconds of CPU time, not several seconds of real time, needs to be se_long, otherwise will timeout sometimes in Azure Pipelines.